src/main.py

```diff
@@ -2716,6 +2717,15 @@ def post_all(self, o):
         email = post["email"]
         signature = post["signature"]
         quicklinks = post["quicklinksid"]
+        rawfavouritereports = post["favouritereportsid"]
+        favouritereports = ""
+        for reportid in rawfavouritereports.split(","):
+            for report in asm3.reports.get_reports(o.dbo):
+                if report.ID == int(reportid):
+                    reporttitle = report.TITLE
+            if favouritereports != "":
+                favouritereports += ","
```

Comment:
this line and the if statement before it can be removed, and replaced later with ",".join(quickreportscfg) to stitch them together